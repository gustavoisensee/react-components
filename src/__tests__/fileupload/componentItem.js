import React from 'react';
import renderer from 'react-test-renderer';
import FileUploadItem from './../../modules/fileupload/componentItem';

describe('FileUploadItem component test', () => {
  test('FileUploadItem component renders', () => {
    const props = {
      item: {
        id: 1,
        text: 'test'
      },
      remove: () => {}
    };

    const tree = renderer.create(
      <FileUploadItem {...props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
