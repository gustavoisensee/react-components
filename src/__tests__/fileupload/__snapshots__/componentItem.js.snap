exports[`FileUploadItem component test FileUploadItem component renders 1`] = `
<div
  className="content__item">
  <div
    className="content__item__title">
    <span>
      test
    </span>
  </div>
  <div
    className="content__item__buttons">
    <button
      className="btn-remove"
      onClick={[Function]} />
  </div>
</div>
`;
